portfolio/src/main/webapp/src/com.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// File Modified by Cody Rivera June 2020
+
+/**
```

Comment:
nit: `com` is most often used to shorthand "communications", in this case I'd prefer writing out the full word i.e. `comments.js`

Reply:
To be fair, this page does need to communicate more extensively with the server.

Nevertheless, I will change it in one of my next few pull requests.